src/login/login.component.tsx

```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import { performLogin } from "./login.resource";
 import { always } from "kremling";
-import styles from "./login.component.css";
 import { getCurrentUser } from "@openmrs/esm-api";
 import { useConfig } from "@openmrs/esm-module-config";
 import { Trans } from "react-i18next";
+import styles from "../styles.css";
+
+const NEXT_PAGE = "/login/location";
```

Comment:
(sharing this even though I don't expect you to change this because it falls into personal preference)

I honestly don't think consts like this have any value. The java community is a big fan of trying to feel better about magic strings by making them all caps at the top of the file, but I don't think it provides any value. Since this const is not even being used in two places, I personally would just inline it into line 53. Consts like this feel like introducing a bit of abstraction for a future-proofed system that will never come to fruition.

Reply:
That's fair. I like consts like this as a way of highlighting the "interface" of a component -- how it interacts with the outside world. But totally agree that it's a matter of personal preference. This code will probably change quite soon anyway (e.g. for [MF-111](https://issues.openmrs.org/browse/MF-111)).